staging/src/kubevirt.io/api/core/v1/schema.go

```diff
@@ -363,8 +362,7 @@ type Realtime struct {
 // NUMAGuestMappingPassthrough instructs kubevirt to model numa topology which is compatible with the CPU pinning on the guest.
 // This will result in a subset of the node numa topology being passed through, ensuring that virtual numa nodes and their memory
 // never cross boundaries coming from the node numa mapping.
-type NUMAGuestMappingPassthrough struct {
-}
```

Comment:
Thanks for refactoring!
It would be easier to review if you'd have a separate commit for this refactoring.